README.md

```diff
@@ -1 +1,198 @@
-# portfolio-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  
+
+  <h3><b>MY README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
```

Comment:
- [ ] Kindly make sure to add these relevant sections `Show your support` and `Acknowledgments` to your README, to make it look professional and according to the Microverse README template. 😀

![ss](https://user-images.githubusercontent.com/105105262/227187206-f650c977-5132-4312-be9f-53dd373845de.PNG)

![ackn](https://user-images.githubusercontent.com/105105262/227187249-e076bfda-2a64-4899-9f3c-d8279f818b40.PNG)

- In your `Show your support` section, you can write something like that:
  > Please give a ⭐️if you like this project

- And in your `Acknowledgments` section, you can give credits to Microverse by thanking them. 😊